styles2.css

```diff
@@ -1,211 +1,288 @@
-@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
-
-body {
-  background-color: #fff;
+.form-list li {
+  list-style: none;
 }
 
-#bg-work1 {
+.about-main {
+  background-image: url("./Image/header-shapes-mobile.png");
   background-repeat: no-repeat;
-  box-sizing: border-box;
-  display: flex;
-  display: grid;
-  flex-direction: column;
-  align-items: center;
-  padding: 16px;
-  gap: 12px;
-  width: 327px;
-  height: 680px;
+  background-position: right;
+  background-size: 100%;
+  background-attachment: unset;
+  background-position-x: 9px;
+  background-position-y: -6px;
   background-color: #fff;
-  opacity: 20px;
-  border: 1px solid #dfe1e6;
-  border-radius: 16px;
-  margin-top: 170px;
-  justify-content: space-between;
+  border-radius: 0 0 0 40px;
+  font-family: "Poppins", sans-serif;
+  width: 100%;
 }
 
-#work-container {
-  width: 295px;
-  height: 220px;
-  margin-bottom: 40px;
-  border-radius: 8px;
-}
+@media screen and (min-width: 768px) {
+  .menu-icon {
+    display: none;
+  }
 
-.work-content h2 {
-  width: 295px;
-  height: 36px;
-  font-family: "Poppins", sans-serif;
-  font-weight: 900;
-  font-size: 32px;
-  line-height: 44px;
-  color: #172b4d;
-}
+  .header {
+    /* opacity: 0; */
+    background: rgba(76, 175, 80, 0.1);
+    position: fixed;
+    width: 100%;
+    box-shadow: rgba(0, 0, 0, 0.1) 0 4px 12px;
+    z-index: 6;
+  }
```

Comment:
- [ ] I like that you're trying to add your own flavor to the navbar here but I'd advise you to stick to the Figma design at least at this stage as the objective of this module is to get you familiar with how to work with a given design and this will ensure your future clients that you can implement a given design down to the last pixel. 🎯
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/230641794-bc5fa83e-e5b8-46dc-8f0a-b8039cbb13e5.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230641897-127f3466-bd85-4aac-9a42-adabbaa3437e.png).
    - You can just fix the background color according to the design that would be enough.
- [ ] Also please make sure the navlinks have a `#344563` color as specified in the [Figma design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=5CSJaDfGx3044Nab-0)